components/navbar/sidebar.snip.html

```diff
@@ -17,44 +17,46 @@
 }}
 
 <!-- Start Sidebar -->
-<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}}" layout="nodisplay">
+<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} {{sidebarclasses}}" layout="nodisplay">
   <div class="flex justify-start items-center ampstart-sidebar-header">
     <div role="button" aria-label="close sidebar" on="tap:{{sidebar-id}}.toggle" tabindex="0" class="ampstart-navbar-trigger items-start">✕</div>
   </div>
-  <nav class="ampstart-sidebar-nav ampstart-nav">
-    <ul class="list-reset m0 p0 ampstart-label">
-      {{#nav}}
-        {{#link}}
-          <li class="ampstart-nav-item"><a href="{{{url}}}">{{text}}</a></li>
-        {{/link}}
-        {{#dropdown}}
-          <li class="ampstart-nav-item ampstart-nav-dropdown relative">
-            {{> ../dropdown/dropdown-inline.snip.html}}
-          </li>
-        {{/dropdown}}
-        {{#search}}
-          <li  class="ampstart-nav-item ampstart-nav-search">
-            <input type="text" placeholder="{{text}}"></input>
-          </li>
-        {{/search}}
-      {{/nav}}
-    </ul>
-  </nav>
+  <div class="ampstart-sidebar-container">
```

Comment:
Good shout! We had it in initially because we were aligning the whole container to the bottom but the design changed. Removed container and related css.